app/styles/all.scss

```diff
@@ -21,8 +21,125 @@ $color_nhsuk-green--shadow: #00401e;
   float: right;
   font-weight: 600;
   padding: 6px 8px;
+  margin-right: -24px !important;
+  margin-top: -4px !important;
 }
 
 .bcc-body {
   height: 100% !important;
 }
+
+.code-wrapper {
+  .code-title {
+    background: #191919;
+    color: #fff;
+    padding: 20px 35px;
+    margin-bottom: 0;
+    z-index: 12;
+    border-radius: 5px 5px 0 0;
+    position: relative;
+  }
+  .code-block {
+    background: #222;
+    color: #fff;
+    font-size: 14px;
+    font-family: monospace;
+    overflow: scroll;
+    max-height: 500px;
+    padding: 26px;
+    width: 100%;
+
+    /* width */
+    &::-webkit-scrollbar {
+      width: 8px;
+      height: 8px;
+    }
+
+    /* Track */
+    &::-webkit-scrollbar-track {
+      border-radius: 10px;
+      background: #191919;
+    }
+
+    /* Handle */
+    &::-webkit-scrollbar-thumb {
+      background: rgba(255, 255, 255, 0.2);
+      border-radius: 10px;
+    }
+
+    /* Handle on hover */
+    &::-webkit-scrollbar-thumb:hover {
+      background: rgba(255, 255, 255, 0.4);
+    }
+
+    /* Corner */
+    &::-webkit-scrollbar-corner {
+      background: rgba(0,0,0,0);
+    }
+
+    .json-text-input {
+      background: none;
+      border: none;
+      outline: none;
+      color: #ce9075;
+      font-size: 14px;
+      font-family: monospace;
+      white-space: pre;
+      border-bottom: 1px dashed rgba(255, 255, 255, 0.3);
```

Comment:
please change to nhs colours if they already exist

Reply:
They don't have a colour for this